qa/rpc-tests/test_framework/util.py

```diff
@@ -311,16 +303,70 @@ def initialize_chain(test_dir, num_nodes, cachedir):
         stop_nodes(rpcs)
         wait_bitcoinds()
         for i in range(MAX_NODES):
+            # record the system time at which the cache was regenerated
+            with open(log_filename(cachedir, i, 'cache_config.json'), "w", encoding="utf8") as cache_conf_file:
+                cache_config = { "cache_time": time.time() }
```

Comment:
Previously this was just using the filesystem timestamp on the cache; I guess perhaps I should have done that after block generation rather than before, but it's a small enough delta that I don't think it matters.